core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -353,6 +372,32 @@ public void onSwipeBottom() {
     new ItemTouchHelper(tabCallback).attachToRecyclerView(tabRecyclerView);
   }
 
+  public Animation transitionRight() {
```

Comment:
So I recently made some animation addition to this [file too](https://github.com/kiwix/kiwix-android/pull/1706/files) and I'd prefer if we are using xml to perform these animations as the Animation constructors are a bit bulky and the xml better describes the animation I find.

Sorry I didn't submit this review comment along with my earlier one, got distracted by a sea of notifications

Reply:
@macgills made the changes.
Is it fine now?